ui/angular.json

```diff
@@ -24,7 +24,8 @@
               "node_modules/@clr/icons/clr-icons.min.css",
               "node_modules/@clr/ui/clr-ui.min.css",
               "node_modules/nouislider/distribute/nouislider.min.css",
-              "src/styles.css"
+              "src/styles.css",
+              "src/assets/scss/styles.scss"
```

Comment:
If we're going to be using scss, we should configure and use CSS Modules so we maintain local scoping of class names.

https://github.com/webpack-contrib/css-loader#modules

This will probably require some tinkering with the webpack config, so it probably deserves its own PR, but we should tackle that before we accrue too many global class names.

Reply:
Yup - this was actually my exact worry with scss, because I couldn't figure out the webpack config tinkering required. We will definitely be discussing further at our tech talk the day we come back from break. But if you have any ideas about how to get this setup correctly that would be great! (Feel free to commit to this branch, as I'm now treating this branch as the scss demo)